### PR TITLE
Added more documentation based on the previous one

### PR DESCRIPTION
**Pull request by jbadiapa**
It generates a new Documentation.txt file, which has:
- Readme.md
- Actions (Tasks of the role)
- Configuration

And which does not have:
* Integration with tripleo
* Contributing
* License

### CODE REVIEW

**Review by mrunge — APPROVED** [review]
cool!

File: Makefile [hunk 1]
```diff
@@ -17,5 +17,9 @@ README.html: README.md docs/metadata.yml
 	docs/metadata.yml README.md -o $@ \
 	--standalone --toc --toc-depth 2
 
+Documentation.txt: README.md docs/metadata.yml roles/*
```
Comment (larsks) [inline Makefile]: I don't understand why we're including `README.md` in another document.  I don't think we need to duplicate the content.

Comment (larsks) [reply Makefile]: Also, depending on `roles/*` is probably not what you want (you want to depend on individual _files_ inside the roles directory; see the `DEFAULTS` variable at the top of the `Makefile`).

File: tools/documentator.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                    print "      * ",b.name
+            print
+
+
+        if os.path.isfile(DEFAULTS):
+            print('\n')
+            print('### Configuration')
+            print('\n\n')
+
+            with open(DEFAULTS) as fd:
```
Comment (larsks) [inline tools/documentator.py]: This seems to be doing the same thing here as `tools/extract-role-docs.py`...the output of which is already included in `README.md`.

**Review by larsks — CHANGES_REQUESTED** [review]
I think we should simplify this, and instead of generating a new file and performing a lot of duplicate work, this should _just_ generate the Ansible action documentation, and should generate new section of the `README.md` file named something like "Playbook outline". Does that make sense?

**Comment by mrunge** [comment]
PLEASE DO NOT MERGE. THIS WILL BREAK REPLICATION.

PLEASE PROPOSE A NEW PATCH VIA GIT REVIEW.

**Comment by larsks** [comment]
Moved to:
- http://review.rdoproject.org/r/3787
- http://review.rdoproject.org/r/3788
- http://review.rdoproject.org/r/3789

**Pull request closed without merging**